src/grading/views.py

```diff
@@ -13,6 +13,56 @@
 import models
 
 
+class ResponseExportView(MethodView):
+    decorators = [login_required, auth.permissions_author]
+
+    def get(self, response_id):
+        response = models.TaskResponse.query.filter_by(id=response_id).first()
+        xml_data = json.loads(response.task.xml_data)
+        course = response.task.course
+        if course not in current_user.get_courses_where_teacher_or_ta():
+            return "Permission Denied", 401
+        filename = 'src/static/uploads/response_%s.xml' % response_id
```

Comment:
should we make folders for each course so that response IDs don't collide with each other? or are responseIDs unique across all tasks and courses?